src/resources/CaseAssistConfigs/CaseAssistPreviewInterfaces.ts

```diff
@@ -0,0 +1,40 @@
+import {New} from '../BaseInterfaces';
+import {CaseAssistConfigModel} from './CaseAssistConfigInterfaces';
+
+export interface Document {
+    title: string;
+    clickUri: string;
+    excerpt: string;
+    fields: string;
```

Comment:
@nathanlb Are you sure `fields` really is a `string`? I would have expected something like `{[key: string]: string}`.